Src/Microsoft.Dynamic/Runtime/CompilerContext.cs

```diff
@@ -22,46 +22,13 @@ namespace Microsoft.Scripting.Runtime {
     /// from this class to provide additional contextual information.
     /// </summary>
     public sealed class CompilerContext {
+        public SourceUnit SourceUnit { get; }
 
-        /// <summary>
-        /// Source unit currently being compiled in the CompilerContext
```

Comment:
Sure. Should we follow StyleCop rules for such comments? "Gets or sets etc..." http://stylecop.soyuz5.com/SA1623.html